server/service/src/sync/test/pull_and_push.rs

```diff
@@ -113,7 +109,7 @@ async fn test_sync_pull_and_push() {
     });
 
     // Test ids and table names
-    assert_eq!(
+    pretty_assertions::assert_eq!(
```

Comment:
how much is the cost of leaving these in? thought we just used in dev, but its way more helpful when things fail, would save us a round of compile/run to leave in...

Reply:
[d0dafc9](https://github.com/msupply-foundation/open-msupply/pull/6587/commits/d0dafc95167f4102e7bbf911c1f6e603b5ddf193)